extras/paid-courses.md

```diff
@@ -86,6 +86,10 @@
 ### Artificial Intelligence
 
 ### Machine Learning
+Name | Provider
+:-- | :--:
+[Machine Learning by Andrew Ng](https://www.coursera.org/learn/machine-learning) | [Coursera](https://www.coursera.org)
+[Machine Learning Engineer Nanodegree](https://www.udacity.com/course/machine-learning-engineer-nanodegree--nd009) | [Udacity](https://www.udacity.com)
```

Comment:
I believe Nanodegree's fall out of the scope for these sections and I think fall into [Specializations](https://github.com/open-source-society/computer-science#specializations) but @ericdouglas would have to make the final call.